src/contracts/interfaces/IPaymentCoordinator.sol

```diff
@@ -0,0 +1,173 @@
+// SPDX-License-Identifier: BUSL-1.1
+pragma solidity =0.8.12;
+
+import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import "./IStrategy.sol";
+
+/**
+ * @title Interface for the `IPaymentCoordinator` contract.
+ * @author Layr Labs, Inc.
+ * @notice Terms of Service: https://docs.eigenlayer.xyz/overview/terms-of-service
```

Comment:
Probably some context as to what role this plays in the protocol. It'd be nice to be able to read the top of a file to understand what it does. Just a nit-pick.